app/controllers/group-chats.js

```diff
@@ -0,0 +1,75 @@
+const messageModel = require('../models/message');
+
+const messages = {
+
+    getOneMessage: (req, res) => {
+        let userID = req.body.user.sourceID;
+
+        messageModel.findBySourceID(userID, (error, results) => {
+            if (error) {
+                throw error;
+            } else {
+                res.status(200).send(results);
+            }
+        });
+    },
+
+    createMessage: (req, res) => {
+        let incomingMessage = req.body.message;
+        let newMessage = new messageModel(incomingMessage);
+
+        if (incomingMessage) {
+            newMessage.save((error, results) => {
+                if (error) {
+                    throw error;
+                } else {
+                    res.status(200).send(results);
+                    //I should emit the message at this point using socket.io
+                }
+            });
+        } else {
+            res.status(500).send('you cannot send an empty message');
+        }
+
+    },
```

Comment:
A function for creating a new group chat. The request should have the message with the different appropriate fields like: sourceID, recepientID, body and messageType. messageType was added in anticipation of other type of messages we might want to implement.